src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -10981,8 +10981,14 @@ public override void VisitForEachIterationVariables(BoundForEachStatement node)
 
         public override BoundNode? VisitObjectInitializerMember(BoundObjectInitializerMember node)
         {
-            // Should be handled by VisitObjectCreationExpression.
-            throw ExceptionUtilities.Unreachable();
+            // We reach this point when we have just an identifier name in the initializer without an actual
```

Comment:
>// We reach this point when we have just an identifier name in the initializer without an actual [](http://example.com/codeflow?start=12&length=96)

Adjustments to this function likely won't be necessary if we create ```BoundAssignmentOperator``` as suggested.

Reply:
Reverted